client/jetpack-connect/plans-landing.jsx

```diff
@@ -92,6 +99,11 @@ class PlansLanding extends Component {
 					onSelect={ this.storeSelectedPlan }
 				>
 					<PlansSkipButton onClick={ this.handleSkipButtonClick } />
+					<LoggedOutFormLinks>
+						<JetpackConnectHappychatButton>
+							<HelpButton onClick={ this.handleHelpButtonClick } />
```

Comment:
The text is probably inaccurate:

> Get help connecting your site

The site should be connected at this point. Should it read something like "Get help selecting your plan"?

![button](https://user-images.githubusercontent.com/841763/31331126-823b04ee-ace1-11e7-9c0c-bd8398444732.png)

Reply:
Makes sense to me. Will address separately, see #18659.